cgi-bin/ljtestlib.pl

```diff
@@ -0,0 +1,83 @@
+package LJ;
+
+use strict;
+no warnings 'uninitialized';
+
+
+BEGIN {
+    use Test::MockObject;
+
+    my $mock = Test::MockObject->new();
+
+    sub fake_lang_ml {
+	my ( $code, $vars ) = @_;
+	my $aopts = $vars->{'aopts'};
+	if ( $code == "cleanhtml.error.markup.extra" ) {
```

Comment:
Unfortunately, Perl uses different comparators for string vs numerical comparison. You'll want to use `eq` here.

See https://perldoc.perl.org/perlop.html#Equality-Operators for more information.

Reply:
done